manifests/params.pp

```diff
@@ -31,11 +31,16 @@
       }
     }
     'Debian': {
+      $default_su_user = versioncmp($::operatingsystemmajrelease, '14.00') ? {
```

Comment:
can you please use the new $facts[] hash instead of the topscope variable?

Reply:
Commit updated. Switched to $facts[] in both the line I added and the one for default su group that was already there.